travis/travis_install_nightly

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-pip install -q QUnitSuite flake8 coveralls pylint Click > /dev/null 2>&1
+pip install -q QUnitSuite flake8 coveralls Click oca-pylint-plugin > /dev/null 2>&1
```

Comment:
The pylint plugin project should move to OCA before we merge this.
I suggest  you to open a discussion on the Community ML.